app/scripts/actions/index.js

```diff
@@ -312,6 +315,18 @@ export const removeGranule = (granuleId) => ({
   }
 });
 
+export const recoverGranule = (granuleId) => ({
```

Comment:
Now that we have `config` in the store, it would be cool to re-factor this action (and later the other actions) to read it from the store rather than reading the value read from the file (`_config`). This further isolates our actions to only interact with the store instead of side effects.

Completed untested refactor:

```
export const recoverGranule = (granuleId) => {
  return (dispatch, getState) => {
    const { config } = getState();
    return dispatch({
      type: types.GRANULE_RECOVER,
      method: 'PUT',
      id: granuleId,
      path: `${config.recoveryPath}/${granuleId}`,
      body: {
        action: 'recoverGranule'
      }
    });
  };
};
```